.github/workflows/volume_group_operator_ci.yaml

```diff
@@ -0,0 +1,98 @@
+
+name: volume-group-operator-ci
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+    types: 
+      - opened
+      - reopened
+  workflow_dispatch:
+    inputs:
+      production:
+        description: 'Click the checkbox for production images'
+        default: false
+        type: boolean
+        required: false
+env:
+  IMAGE_VERSION: '1.11.0'
+jobs:
+  prepare_env: 
+    name: "Prepare env"
+    runs-on: ubuntu-latest
+    env:
+      PRODUCTION: "${{ inputs.production }}"
+      BUILD_NUMBER: "${{ github.run_number }}"
+      PROD_REPOSITORY: volume-group-operator
+      STAGING_REPOSITORY: volume-group-operator-staging
+    steps:
+       - name: Checkout code
+         uses: actions/checkout@v3.2.0
+       - uses: FranzDiebold/github-env-vars-action@v2.7.0
+       - name: Setup dependencies
+         id: setup_dependencies
+         run: |
+           build/ci/github_actions/setup_ci_dependencies.sh
```

Comment:
should the next command work?
docker pull quay.io/csiblock/volume-group-operator

Reply:
with v1 tag it works

Reply:
we don't have latest tag in `quay.io/csiblock/volume-group-operator`, we can create one, but `docker pull quay.io/csiblock/volume-group-operator-staging` will work

Reply:
@matancarmeli7 , please open a ticket to trigger the automation tests on every volume group code change on production, ok?

Reply:
opened CSI-5162

Reply:
thanks